video_store.py

```diff
@@ -0,0 +1,147 @@
+import requests
+import datetime
+
+# - Customer create, edit, delete  --- check
+# - Video create, edit, delete  --- check 
+# - List customers --- check
+# - List videos --- check
+# - Check out video to customer --- *missing
+# - Check in video  --- *missing
+
+class VideoStore:
+    def __init__(self, url="https://retro-video-store-api.herokuapp.com", selected_video=None, selected_customer=None, selected_rental=None):
+        self.url = url
+        self.selected_video = selected_video
+        self.selected_customer = selected_customer
+        self.selected_rental = selected_rental # operate through this attribute or video?
+
+    def create_customer(self, name="Default Customer Name", postal_code="Default Postal Code", phone="Default Phone Number",\
+                        register_at=str(datetime.datetime.now()), videos_checked_out_count=0): 
+        relevant_params = {
+            "name": name,
+            "postal_code": postal_code,
+            "phone": phone,
+            "register_at": register_at,
+            "videos_checked_out_count": videos_checked_out_count
+        }
+        response = requests.post(self.url + "/customers", json=relevant_params)
```

Comment:
It's a best practice to check the status code before you use a response.  (Even if the API generally gives well formed responses.)